cmd/crl-updater/main.go

```diff
@@ -51,6 +51,12 @@ type Config struct {
 		// recommend an UpdatePeriod of 6 hours.
 		UpdatePeriod cmd.ConfigDuration
 
+		// UpdateOffset controls the times at which crl-updater runs, to avoid
+		// scheduling the batch job at exactly midnight. The updater runs every
+		// UpdatePeriod, starting from the Unix Epoch plus UpdateOffset, and
+		// continuing forward into the future forever.
+		UpdateOffset cmd.ConfigDuration
```

Comment:
"Should be less than UpdatePeriod."

Reply:
Good call. I've also added code to clamp the value so that a misconfiguration doesn't result in misbehavior.